### ISSUE TITLE
Mounting bug in water

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5440]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Even when swimming in water I am able to mount.
**Expected behaviour**: Tell us what should happen instead

Mounting should not be available while swimming/underwater

**Steps to reproduce the problem**:

1. Find any water body and enter it
2. Try summoning your ground mount
3. 

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/32976781/39520944-0fc274ce-4e3f-11e8-93b6-4f23b45680ca.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
In Wotlk you could mount ground mounts in water. You can't use flying mounts.

**Comment by Golganeth** [comment]
@wrathofthebug Please when reporting an issue provide enough proof in order to backup the problem.
@alexuktc I believe so too, I'm not 100% sure though.

I didn't search very thoroughly, but all I could find was this:
https://wow.gamepedia.com/Mount
> Wrath of the Lich King Patch 3.1.0 (2009-04-14): All Ground Mounts may now swim without dismounting the rider. Flying Mounts still may NOT, and will dismount the rider upon entering water. 

--------------

Since this seems to be the intended behavior, I'm closing the report until any proof that says otherwise is provided.
